packages/prompts/src/index.ts

```diff
@@ -679,16 +686,38 @@ export const outro = (message = '', opts?: CommonOptions) => {
 
 export interface LogMessageOptions extends CommonOptions {
 	symbol?: string;
+	spacing?: number;
+	secondarySymbol?: string;
 }
 export const log = {
 	message: (
-		message = '',
-		{ symbol = color.gray(S_BAR), output = process.stdout }: LogMessageOptions = {}
+		message: string | string[] = [],
+		{
+			symbol = color.gray(S_BAR),
+			secondarySymbol = color.gray(S_BAR),
+			output = process.stdout,
+			spacing = 1,
+		}: LogMessageOptions = {}
 	) => {
-		const parts = [`${color.gray(S_BAR)}`];
-		if (message) {
-			const [firstLine, ...lines] = message.split('\n');
-			parts.push(`${symbol}  ${firstLine}`, ...lines.map((ln) => `${color.gray(S_BAR)}  ${ln}`));
+		const parts: string[] = [];
+		for (let i = 0; i < spacing; i++) {
+			parts.push(`${secondarySymbol}`);
+		}
+		const messageParts = Array.isArray(message) ? message : message.split('\n');
+		if (message && messageParts.length > 0) {
```

Comment:
not needed
```suggestion
		if (messageParts.length > 0) {
```

Reply:
this one is here so when you `message('')`, it won't render blank lines